### PR TITLE
New Steal Objective: AI Upload board

### PR DESCRIPTION
**Pull request by Runian**
# Document the changes in your pull request
The AI Upload board is now a steal objective. Considering that they aren't mass printable due to #6764 and that this is an important board, it makes sense for this to be a steal objective. 

It has a difficulty of 5.
Chief Engineer, Research Director and the Network Admin are prevented from having this as an objective.

:cl:  
rscadd: The AI Upload board is now something that the Syndicate wants to steal.
/:cl:

### CODE REVIEW

**Comment by athena413** [comment]
Holy shit yes.

Suggestion: Increase difficulty to 5 (its very valuable)

Also

Restrict AT LEAST netmin from getting the objective, and probably Station Engineer as well since they are able to get very very close to the board with a lot more ease. Atmos tech, maybe, as well, but they don't have access to tech storage outside of lowpop.

**Comment by Runian** [comment]
I've upped the difficulty to 5. I also restricted it so that the Chief Engineer and the Research Director can't get it since the former has the access (to the airlock) and the latter has the authority (as the caretaker of the AI). 

> Holy shit yes.
> 
> Suggestion: Increase difficulty to 5 (its very valuable)
> 
> Also
> 
> Restrict AT LEAST netmin from getting the objective, and probably Station Engineer as well since they are able to get very very close to the board with a lot more ease. Atmos tech, maybe, as well, but they don't have access to tech storage outside of lowpop.

I'd have to restrict 6 jobs total if I wanted to follow that which is kind of much: Network Admin, Station Engineer, Atmospheric Technician (lowpop), Roboticist, Scientist (lowpop), and Geneticist (lowpop). I am more-or-less matching it to how other theft objectives decide who can get it. For example, full plasma tank excludes everyone that can directly access a plasma tank and can essentially legally carry it. The same goes for slime extract objective, though it does not exclude the Roboticist (who is locked behind a normal glass window).

**Comment by athena413** [comment]
> I'd have to restrict 6 jobs total if I wanted to follow that which is kind of much: Network Admin, Station Engineer, Atmospheric Technician (lowpop), Roboticist, Scientist (lowpop), and Geneticist (lowpop). I am more-or-less matching it to how other theft objectives decide who can get it. For example, full plasma tank excludes everyone that can directly access a plasma tank and can essentially legally carry it. The same goes for slime extract objective, though it does not exclude the Roboticist (who is locked behind a normal glass window).

Scientist roboticist and geneticist would have trouble accessing it quietly most rounds, just like any other role. I say netmin because they have a minor amount of authority over the AI, station engineer because they have direct access to tech storage with little suspicion _on top of_ ready access to hardsuits to break in via space. Atmos tech the same, but with more trouble since they don't have direct access to tech storage, but do have a fire axe to break the window in. Honestly though, it's probably fine, just an easier objective (but still not _literally free_).

**Comment by Runian** [comment]
I'll exclude the Signal Admin as a compromise. I think a different PR should be opened to adjust all the excluded jobs for the steal objectives at the same time so it is all consistent.

**Comment by neeshacark** [comment]
im pretty positive you can still print them, i recall doing it a couple months ago

**Comment by Runian** [comment]
> im pretty positive you can still print them, i recall doing it a couple months ago

#6764

**Comment by neeshacark** [comment]
> > im pretty positive you can still print them, i recall doing it a couple months ago
> 
> #6764

yes im aware that pr exists and im telling you that im pretty confident i printed them literally months ago and thats from 2019

**Comment by Runian** [comment]
> > > im pretty positive you can still print them, i recall doing it a couple months ago
> > 
> > 
> > #6764
> 
> yes im aware that pr exists and im telling you that im pretty confident i printed them literally months ago and thats from 2019

Please show proof that you can currently print out AI Upload boards (and Cyborg Upload boards) as I am also confident that these boards cannot be printed.

**Review by ToasterBiome — APPROVED** [review]

File: code/game/gamemodes/objective_items.dm [hunk 1]
```diff
@@ -83,6 +83,12 @@
 	difficulty = 3
 	excludefromjob = list("Head of Security", "Warden")
 
+/datum/objective_item/steal/aiupload
+	name = "an AI Upload board."
```
Comment (ToasterBiome) [inline code/game/gamemodes/objective_items.dm]:
```suggestion
	name = "an AI upload board."
```

**Review by ToasterBiome — CHANGES_REQUESTED** [review]

**Review by ToasterBiome — APPROVED** [review]

**Pull request merged**